index.html

```diff
@@ -21,6 +21,50 @@
             <a class="back_con" href="#contact"">Contact</a>
         </div>
     </div>
+
+    <div id="display_work">
+        <div class="work_overlay">
+        </div>
+        <div class="center_card">
+            <div class="cardPop">
+                <div class="work_pop_title">
+                    <h4 id="work_pop_name"></h4>
+                    <a class="close_work">&times;</a>
+                </div>
+                <ul>
+                    <li class="worktime"><b>CANOPY</b></li>
+                    <li class="worktime"><img src="images/Counter.png"></li>
+                    <li class="worktime worktitle">Back End Dev</li>
+                    <li class="worktime"><img src="images/Counter.png"></li>
+                    <li class="worktime worktitle year"></li>
+                </ul>
+                <div>
+                    <img class="imageWork" alt="SnapshootPortfolio">
+                </div>
+                <div class="workInfoDesktop">
+                    <div>
+                        <p class="workDetail"></p>
+                    </div>
+                    <div class="techLiveSource">
+                        <ul class="client techBox">
+                            <li class="worktime tag tech1"></li>
+                            <li class="worktime tag tech2"></li>
+                            <li class="worktime tag tech3"></li>
+                        </ul>
+                        <div class="buttonBox">
+                            <a class="goLive">
+                                <button class="goButtons" type="button">See Live <img src="images/live.png" alt="Go live"></button>
+                            </a>
+                            <a class="goSource">
+                                <button class="goButtons" type="button">See Source <img src="images/source.png" alt="Go Source"> </button>
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
     <section id="main">
```

Comment:
- [ ] As I indicated earlier, the requirement is trying to strengthen your DOM skills and your creativity by asking you to devise means of loading your `work-section` and your `pop-up` using JavaScript. As a result of that, there should be no project cards content in your `index.html` except for the sectional container that you would like to query.
- please observe the screenshot below, the contents within the red circle are expected to be removed from your `html` file and loaded from your `javascript` file/s when the page loads. That is called dynamic rendering :wink: You will love it by the time you are done. :muscle: